Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -93,6 +117,11 @@ private void OnMapInit(EntityUid uid, SuitSensorComponent component, MapInitEven
             }
         }
 
+        private void OnComponentInit(EntityUid uid, SuitSensorComponent component, ComponentInit args)
+        {
+            component.StationId = _stationSystem.GetOwningStation(uid);
+        }
```

Comment:
why not on mapinit as well?

Reply:
I moved it to mapinit and added a data field annotation